02-oop-inheritance/Observer pattern/script.js

```diff
@@ -0,0 +1,66 @@
+var Movie;
+var MovieObserver;
+
+//metodos para publicar y subscribir
+var EventBus = {
```

Comment:
Remember that the idea is to create a "class", so you should maybe think on having a more descriptive variable like ``Observers`` and then extend their prototypes with your ``subscribe`` and ``publish``methods.